src/nvme/types.h

```diff
@@ -6024,4 +6026,30 @@ enum nvme_zns_report_options {
 	NVME_ZNS_ZRAS_REPORT_OFFLINE		= 0x7,
 };
 
+/**
+ * struct nvme_media_unit_stat_desc  
```

Comment:
- whitespace at the end of the line
- please add some documentation for each member.

Reply:
Addressed
Thanks @igaw